scripts/firedrake-zenodo

```diff
@@ -235,13 +235,18 @@ tag += "." + str(index + 1)
 for component in components:
     repo = gh.repository(projects[component], component)
 
-    commit = repo.commit(shas[component])
-    if not commit:
-        log.error("Failed to find specified commit for %s" % component)
-    shas[component] = commit.sha
+    try:
+        commit = repo.commit(shas[component])
+        if not commit:
+            log.error("Failed to find specified commit for %s" % component)
```

Comment:
I think so. The usual cause of this is that the user has not pushed their commit. In that case we still want a complete set of commits to form a consistent release (and the user should be told to push). If you *really* want to drop the non-pushed commit then you can edit the incoming json to remove it.